Source/DataModels/Resources/ResourceAnimation.h

```diff
@@ -38,7 +38,7 @@ struct Channel
 	double GetDuration() const;
 
 	void SetDuration(double duration);
-	void SetChannels(std::unordered_map<std::string, ResourceAnimation::Channel*> channels);
+	void SetChannels(const std::unordered_map<std::string, ResourceAnimation::Channel*> channels);
```

Comment:
Whoops, this is still wrong, my bad.
```suggestion
	void SetChannels(const std::unordered_map<std::string, ResourceAnimation::Channel*>& channels);
```